.env.sample

```diff
@@ -18,3 +18,6 @@ PUSHER_APP_ID=YOUR_APP_ID
 PUSHER_APP_KEY=YOUR_APP_KEY
 PUSHER_APP_SECRET=YOUR_APP_SECRET
 PUSHER_APP_CLUSTER=YOUR_APP_CLUSTER
+NEW_USER_PASSWORD=default password of a new user
+FACEBOOK_SDK = 'string'
```

Comment:
Good job on including the new environment variables to the .env.sample file to indicate new environment variables.